app/components/forms/wizard/basic-details-step.js

```diff
@@ -121,6 +121,20 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   },
 
   getValidationRules() {
+    window.$.fn.form.settings.rules.checkMax = (value, min_price) => {
+      let returnValue = false;
+      if (value > min_price) {
+        returnValue = true;
+      }
+      return returnValue;
+    };
+    window.$.fn.form.settings.rules.checkMin = (value, max_price) => {
+      let returnValue = false;
+      if (max_price > value) {
+        returnValue = true;
+      }
+      return returnValue;
+    };
```

Comment:
@niranjan94 @CosmicCoder96 @uds5501 @shreyanshdwivedi I’ve written 2 validation rules checkMax and checkMin as multiple paramters are not supported by validation rules.
Reference:https://github.com/Semantic-Org/Semantic-UI/issues/3682#issuecomment-184583162

It’s getting validated but it always returns false. Always prompts Ticket max price should be greater than min price.

Reply:
@niranjan94 I'm a little stuck in implementing this check for a specific donation ticket. How should I be doing this?

Reply:
@mrsaicharan1 If you are not able to do this with `Semantic UI` , I just you remove the FE validations and throw a server error when `Max Price` is less than `Min Price` , Similar to what we have done in case of `Date & Time` , `Min and Max Number of Tickets orderable at one time`

Reply:
@kushthedude https://github.com/fossasia/open-event-frontend/pull/3194#issuecomment-505288106

Reply:
@kushthedude I had already written a validation check for max and min price! I had completely forgotten about it! Thanks for that. There was a naming convention error in the FE. That was the main reason the validation wasn't able to detect the data attributes going in properly.
![Screenshot 2019-07-02 at 9 00 44 AM](https://user-images.githubusercontent.com/25197147/60480672-08590000-9ca8-11e9-90a1-22743f6ca6b8.png)